### ISSUE TITLE
Can't build

### ISSUE THREAD

**Issue by louisrc3ys** [contour-terminal/contour#849]
Excuse me for being a computer novice, I seem unable to handle this issue.
![2022-10-19_10-40](https://user-images.githubusercontent.com/71259902/196584997-96b19e79-4a3c-414d-ac67-e6f2ad58a593.png)

**Comment by louisrc3ys** [comment]
> 

Yeah, I have run the script and the top lines are the results after the run, My OS: Slackware Kerneal: 5.15.63

**Comment by louisrc3ys** [comment]
I tried to rerun the script：
![2022-10-19_11-15](https://user-images.githubusercontent.com/71259902/196589334-2148841d-c9c8-46c2-a30f-f2f0d610f208.png)

**Comment by Utkarsh-khambra** [comment]
Sorry  we need to add slackware support for installing dependency.
In the meanwhile either you can install dependencies manually by looking at what install_deps.sh is doing or try the [latest release Appimage](https://github.com/contour-terminal/contour/releases/tag/v0.3.4.223).

**Comment by louisrc3ys** [comment]
The AppImage also lacks some dependencies, 
![2022-10-19_11-38](https://user-images.githubusercontent.com/71259902/196592359-841dcbc2-b186-46a0-8704-d542c0ce6e42.png)


 Anyway, thank you for your response, I will try to continue the problem because I want to use this high performance terminal.

**Comment by Yaraslaut** [comment]
Then you need to change install-deps.sh a bit and add the following calls :
```
fetch_and_unpack_Catch2
fetch_and_unpack_fmtlib
fetch_and_unpack_gsl
fetch_and_unpack_yaml_cpp
fetch_and_unpack_range
```
You can add this right after the line with `439: echo "OS $ID not supported."`

**Comment by louisrc3ys** [comment]
> Then you need to change install-deps.sh a bit and add the following calls :
> 
> ```
> fetch_and_unpack_Catch2
> fetch_and_unpack_fmtlib
> fetch_and_unpack_gsl
> fetch_and_unpack_yaml_cpp
> fetch_and_unpack_range
> ```
> 
> You can add this right after the line with `439: echo "OS $ID not supported."`

Thank you! I'll try it.
